client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -18,6 +18,8 @@ import {
   DisplayTableCopyCsv,
   DisplayTableDownloadCsv,
   DisplayTableColumnToggle,
+  PageSelector,
+  SelectPageSize,
```

Comment:
Guess, for consistency, either
  - `PageSelector` should be `SelectPage` or
  - `SelectPageSize` should be `PageSizeSelector`
  
I lean towards `Select...`, but I don't think we have a hard rule on how to name React components so either would be fine. Your choice.